<launch>

<!--     Publish saved trajectory -->
    <node name="end_effector_calibration_ui_node"
        pkg='qlin_calibration' output="screen"
        ns="qlin_calibration"
        type="end_effector_calibration_ui_node.py" required="true" >
        <rosparam file="$(find qlin_calibration)/config/end_effector_calibration_ui.yaml"
            command="load" subst_value="true"/>
    </node>


</launch>
